src/styles/primeng-theme/components/_multiselect.scss

```diff
@@ -0,0 +1,158 @@
+#multiselect-item {
+    margin: 0;
+    padding: $basic-padding;
+
+    &:hover {
+        @extend #ui-hover-state;
+    }
+}
+
+.ui-multiselect {
+    @extend #inputtext-normal-border;
+    border-radius: 0;
+
+    &.ui-state-focus {
+        @extend #inputtext-active-border;
+    }
+
+    &:not(.ui-state-disabled) {
+        &:hover,
+        &:active {
+            @extend #field-hover-state;
+        }
+
+        &.ui-state-focus {
+            .ui-multiselect-trigger {
+                border-color: $dropdown-active-color;
```

Comment:
We talked about create ```$multiselect``` variables and just do like ```$multiselect: $dropdown```` right?

Reply:
you told me to keep the same reference of "$dropdown...".